internal/pkg/manifest/backend_svc.go

```diff
@@ -126,9 +126,20 @@ func (s *BackendService) BuildArgs(contextDir string) map[string]*DockerBuildArg
 	return buildArgs
 }
 
-// EnvFile returns the location of the env file against the ws root directory.
-func (s *BackendService) EnvFile() string {
-	return aws.StringValue(s.TaskConfig.EnvFile)
+// EnvFiles returns the locations of all env files against the ws root directory.
+// This method returns a map[string]string where the keys are container names
+// and the values are either env file paths or empty strings.
+func (s *BackendService) EnvFiles() map[string]string {
+	envFiles := make(map[string]string)
```

Comment:
How about this?
plus 2 for main container as well as FirelensContainer. 
```suggestion
	envFiles := make(map[string]string, len(s.Sidecars)+2)
```

Reply:
What's the behavior of a map initialized with a larger length if it's not actually filled out? 

Like, if I'm iterating over it later am I going to have two iterations of my for loop even if there's not actually an env file for the main container or logging sidecar?

Reply:
The length is "approximate" anyway, and more space will be created if necessary. No, it shouldn't iterate over empty spaces. https://go.dev/ref/spec#Making_slices_maps_and_channels

Reply:
Shipped.

Reply:
Upon further review, this change represented a magic number with little documentation in the code, and has been removed in favor of readability. The ruling on the field is overturned.